From b0b30bf0f004390617d0fbf2710ea6cde9699f57 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 22 Oct 2026 13:10:12 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh gpgkey-6690D8BC.gpg from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 yubikey-manager-qt.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/yubikey-manager-qt.spec b/yubikey-manager-qt.spec
index 216547a..d013246 100644
--- a/yubikey-manager-qt.spec
+++ b/yubikey-manager-qt.spec
@@ -11,7 +11,7 @@ Patch1: yubikey-manager-qt-1.2.5-remove-cloud-upload.patch
 
 License: BSD-2-Clause
 
-BuildRequires: gnupg2
+BuildRequires: openpgpverify
 BuildRequires: gcc-c++
 BuildRequires: make
 BuildRequires: python3
@@ -30,7 +30,7 @@ ExcludeArch:   %{ix86}
 Cross-platform application for configuring any YubiKey over all USB interfaces.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %setup -q -n %{name}
 %patch 1 -p1
 sed -i 's|python |python3 |g' ykman-cli/ykman-cli.pro
-- 
2.55.0

